examples/simple-generator/src/app/index.ts

```diff
@@ -0,0 +1,142 @@
+import { join } from 'path';
+import Generator from 'yeoman-generator';
+
+import type { Template, LROPSettings, OVPSettings } from '@sap-ux/fiori-elements-writer';
+import { generate as generateFE, OdataVersion, TemplateType } from '@sap-ux/fiori-elements-writer';
+import type { OdataService } from '@sap-ux/odata-service-writer';
+import type { Template as FreestyleTemplate } from '@sap-ux/fiori-freestyle-writer';
+import { generate as generateUI5, TemplateType as FreestyleTemplateType } from '@sap-ux/fiori-freestyle-writer';
+import type { Ui5App } from '@sap-ux/ui5-application-writer';
+import { isAppStudio } from '@sap-ux/btp-utils';
+import type { ServiceInfo } from './service';
+import { getServiceInfo, getServiceInfoInBAS } from './service';
+
+export default class extends Generator {
+    private app!: Ui5App & { app: { flpAppId: string } };
+    private service!: OdataService;
+    private template: {
+        type: TemplateType | FreestyleTemplateType.Basic;
+        settings?: LROPSettings | OVPSettings | {};
+    };
+
+    initializing(): void {
+        this.log(
+            'Example of a simple Fiori elements for OData v4 generator that only creates listreport objectpage applications.'
```

Comment:
This is not entirely accurate, we can also generate OVP and freestyle apps...

Reply:
Fixed. https://github.com/SAP/open-ux-tools/pull/466/commits/2d2c2a4316c8a4eca839b5f0ccefc4dc13f44de8